Components/Queues/QueueProducer.php

```diff
@@ -2,6 +2,7 @@
 
 namespace Smartbox\Integration\FrameworkBundle\Components\Queues;
 
+use Smartbox\Integration\FrameworkBundle\Components\Queues\Drivers\AsyncQueueDriverInterface;
```

Comment:
The use of this interface is not needed in this class

Reply:
Ok, it was removed.